presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                     +                                          +                                            +thread to increase CPU utilization with the cost of +
++                                     +                                          +                                            +increased memory usage.                             +
++-------------------------------------+------------------------------------------+--------------------------------------------+----------------------------------------------------+
++   ``task_hash_build_concurrency``   +               ``Integer``                +    ``task.default-concurrency`` (``1``)    +**Experimental.** Default number of local parallel  +
++                                     +                                          +                                            +hash build jobs per worker. Same as                 +
++                                     +                                          +                                            +``task_join_concurrency`` but it is used for        +
++                                     +                                          +                                            +building hashes. The value is always rounded down   +
++                                     +                                          +                                            +to the power of 2 so it's recommended to use such   +
++                                     +                                          +                                            +value in order to avoid unexpected behavior.        +
++-------------------------------------+------------------------------------------+--------------------------------------------+----------------------------------------------------+
++  ``task_aggregation_concurrency``   +               ``Integer``                +    ``task.default-concurrency`` (``1``)    +**Experimental.** Default number of local parallel  +
++                                     +                                          +                                            +aggregation jobs per worker. Same as                +
++                                     +                                          +                                            +``task_join_concurrency`` but it is used for        +
++                                     +                                          +                                            +aggregation.                                        +
++-------------------------------------+------------------------------------------+--------------------------------------------+----------------------------------------------------+
++        ``task_writer_count``        +               ``Integer``                +       ``task.writer-count`` (``1``)        +Describes how many parallel writers may try to      +
```

Comment:
Comment for all copies between session properties and server properties.
I thought it will be easier to read it that way as you don't need to do a lot of jumping while reading docs. However it requires to keep updating two places instead of one, so you're probably right that it should be changed.
However I think that session properties are ones that should contain real description, as they're more important and they are responsible for what will really happen in cluster.